src/components/Header/Header.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { ErrorType } from '../../types/error';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+type Props = {
+  todos: Todo[]
+  addTodo: (newTodo: Omit<Todo, 'id'>) => void
+  setErrorMessage: (message: ErrorType) => void
+};
+
+const USER_ID = 11299;
+
+export const Header:React.FC<Props> = ({
+  todos,
+  addTodo,
+  setErrorMessage,
+}) => {
+  const [newTitle, setNewTitle] = useState('');
+
+  const handerInputTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setNewTitle(event.target.value);
+  };
+
+  const handlerSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setNewTitle('');
+  };
+
+  const hanldlerEnter = (event: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
Fix such typos everywhere
```suggestion
  const handleEnter = (event: React.KeyboardEvent<HTMLInputElement>) => {
```